subprojects/language-groovy/src/main/java/org/gradle/api/tasks/compile/GroovyCompile.java

```diff
@@ -383,4 +385,12 @@ public void setGroovyClasspath(FileCollection groovyClasspath) {
     protected JavaToolChainFactory getJavaToolChainFactory() {
         throw new UnsupportedOperationException();
     }
+
+    /**
+     * Injects and returns an instance of {@link FeaturePreviews}.
```

Comment:
I the Javadoc still necessary? I don't think it adds anything.

Reply:
Removed.